test/dlc_tests/sanity/test_telemetry.py

```diff
@@ -392,7 +392,10 @@ def get_tensorflow_inference_command_tf27_above(image_uri, model_name):
 
     _, image_framework_version = test_utils.get_framework_and_version_from_tag(image_uri)
     if Version(image_framework_version) in SpecifierSet(">=2.7"):
-        inference_command = test_utils.build_tensorflow_inference_command_tf27_and_above(model_name)
+        entrypoint = "/usr/bin/tf_serving_entrypoint.sh"
+        if "neuron" in image_uri:
+            entrypoint = "/usr/local/bin/entrypoint.sh"
```

Comment:
Wondering why we are moving this, instead of keeping the original "copied" name within the docker image.

Reply:
From the first release of neuron dlc for tf we have been using entrypoint.sh as the script (instead of tf_serving_entrypoint.sh). Documentation for say using DLC on inferentia [ECS](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-inference.html) have reference to the entrypoint.sh and hence using that in the test code also to make sure that it gets tested. I could copy the entrypoint.sh  to tf_serving_entrypoint.sh in dockerfile and not modify this test, but that could mean entrypoint.sh never gets tested and that is what the customer could be using.

Reply:
Earlier release was for TF2.5 and this new code takes into effect only for tf27 and above and that did not have the neuron changes.